File: program/program/calculate-compound-interest/CalculateCompoundInterest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
 
public class compoundInterest
{
    static void Main(string[] args)
    {
        float amount, rate, intrest, time, ci, a;
        Console.Write("Enter the principal amount : ");
        amount = Convert.ToInt32(Console.ReadLine());       
 
        Console.Write("Enter the interest rate : ");
        rate = Convert.ToInt32(Console.ReadLine());
 
        Console.Write("Enter the period in years: ");
        time = Convert.ToInt32(Console.ReadLine());        
 
         intrest = 1+(rate/100);
         ci = 1;
         for(a = 1; a <= time; a++)
            ci = ci * intrest;
 
         ci = amount * ci - amount;
 
         Console.WriteLine("Your compound interest is : "+ ci);
 
    }
}

File: program/program/calculate-compound-interest/calculate_compound_interest.dart
import 'dart:io';
import 'dart:math';
void main(){
  print("Enter the principal :");
  double principal = double.parse(stdin.readLineSync()!);
  print("Enter the interest :");
  double interest = double.parse(stdin.readLineSync()!);
  print("Enter the time :");
  double time = double.parse(stdin.readLineSync()!);
  double ci = principal *  pow(1 + interest/100, time);
  print('Area of circle : $ci');
}
